sample/sample.go

```diff
@@ -0,0 +1,181 @@
+package sample
+
+import (
+	"errors"
+	"math"
+	"sort"
+
+	"gonum.org/v1/gonum/floats"
+	"gonum.org/v1/gonum/stat/sampleuv"
+)
+
+type Sampler interface {
+	Sample([]float64) ([]float64, error)
+}
+
+type Temperature float64
```

Comment:
Maybe I'm missing it by if the others have their own init funds, shouldn't this one?

Reply:
I think it's just for the usage as in comment above